astropy/modeling/functional_models.py

```diff
@@ -1432,9 +1432,9 @@ class Box2D(Fittable2DModel):
 
             f(x, y) = \\left \\{
                      \\begin{array}{ll}
-                       A & : x_0 - w_x/2 \\geq x \\geq x_0 + w_x/2 \\\\
-                       A & : y_0 - w_y/2 \\geq y \\geq y_0 + w_y/2 \\\\
-                       0 & : \\textnormal{else}
+                       A & : x_0 - w_x/2 \\leq x \\leq x_0 + w_x/2 \\\\
+                       A & : y_0 - w_y/2 \\leq y \\leq y_0 + w_y/2 \\\\
```

Comment:
Not sure. I can't render this in my head. :sweat_smile: What does this look like?